### DISCUSSION TITLE
Left side add new drawer update!

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5608] [New Feature/Ideas Requests]
Hi, `InspireUI Team`,
Please add the left side drawer like this below ex. *Thanks!*
![alt text](https://i.stack.imgur.com/1UQaq.png "Left Side Drawer Example for Flux store")

**Comment by osman-kocak** [comment]
Just example

![image](https://user-images.githubusercontent.com/57155882/189217561-f43f4ac0-2d43-4e4d-985f-48104efcccd5.png)
